requirements_lock_3_11.txt

```diff
@@ -0,0 +1,1560 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
+#
+#    bazel run //:requirements_3_11.update
+#
+absl-py==2.2.2 \
```

Comment:
Does this need to be made version specific? Most of the time a lockfile generated for an older python version will work with newer versions as well. Although I'm not sure how that interacts with the hermetic python toolchain.